analysis_template/plhld/selection/default.py

```diff
@@ -0,0 +1,164 @@
+# coding: utf-8
+
+"""
+Selection methods for plc2hldr.
+"""
+
+from collections import defaultdict
+from typing import Tuple
+
+from columnflow.util import maybe_import
+from columnflow.columnar_util import set_ak_column
+from columnflow.production.util import attach_coffea_behavior
+
+from columnflow.selection import Selector, SelectionResult, selector
+from columnflow.production.categories import category_ids
+from columnflow.production.processes import process_ids
+from plhld.selection.general import increment_stats, jet_energy_shifts
+from plhld.production.cutflow_features import cutflow_features
+
+np = maybe_import("numpy")
+ak = maybe_import("awkward")
+
+
+@selector(uses={"event"})
+def sel_incl(self: Selector, events: ak.Array, **kwargs) -> ak.Array:
+    # select all
+    return ak.ones_like(events.event)
```

Comment:
can be removed, this is included in `categories.py`

Reply:
I moved it into a single file for the 'cms_minimal' template.